summarization/extractor_output_preprocessor.py

```diff
@@ -268,8 +268,9 @@ def _make_sentence_objects(self):
         # to assign different indices for sentences in paragraph
         sentence_index_in_paragraph = 0
         # step size -based on number of sentences in para
-        step_size_for_sentence_index_in_paragraph \
-            = 1 / len(tokenized_and_cleaned_text_object)
+        if tokenized_and_cleaned_text_object:
+            step_size_for_sentence_index_in_paragraph \
+                = 1 / len(tokenized_and_cleaned_text_object)
```

Comment:
Yes it will!
This is mainly a safeguard for the case when no text contents are extracted from the web page. If that is the case then lists of sentences will be empty and the loop following this statement will not execute. This the step size field will never be used.